src/galileo/__init__.py

```diff
@@ -5,7 +5,10 @@
 
 from galileo.decorator import GalileoDecorator, galileo_context, log
 from galileo.logger import GalileoLogger
+from galileo.resources.models.execution_status import ExecutionStatus
```

Comment:
Why are we using auto-gen schemas instead of the one from `galileo-core`?

Reply:
I pushed the changes for Protect calls to use the galileo-core schemas.

The Stage calls require a bit of extra work as I have to move some schemas from api to core
- https://github.com/rungalileo/core/pull/322
- https://github.com/rungalileo/api/pull/4329
- https://github.com/rungalileo/galileo-python/pull/185